data/settings/src/androidTest/java/com/google/jetpackcamera/settings/LocalSettingsRepositoryInstrumentedTest.kt

```diff
@@ -108,15 +109,15 @@ class LocalSettingsRepositoryInstrumentedTest {
 
     @Test
     fun can_update_default_to_front_camera() = runTest {
-        // default to front camera starts false
-        val initialFrontCameraDefault = repository.getCameraAppSettings().isFrontCameraFacing
-        repository.updateDefaultToFrontCamera()
+        // default to front camera starts as back
```

Comment:
nit: these 2 comments can probably be updated to say default lens facing starts as back, and default lens facing is now front.

Reply:
Thanks for the catch. I fixed these comments and a couple others in the test method below.